File: net/net-registri-log/Logs/Models/Log.cs
using System;

namespace net_registri_log.Logs.Models
{
    public class Log
    {
        public int Id { get; set; }
        public string Message { get; set; }
        public string Level { get; set; }
        public DateTime Timestamp { get; set; }
        public string Exception { get; set; }
        public string UserName { get; set; }
        public string Operation { get; set; }
        public string JsonObject { get; set; }
    }
}

File: node/projects/ngx-registri-log/src/lib/logs/log.ts
export class Log {
    id: number;
    message: string;
    level: string;
    timestamp: Date;
    exception: string;
    userName: string;
    operation: string;
    jsonObject: string;
    jsonObjectObject: object;
    
    constructor(data?: Partial<Log>) {
        Object.assign(this, data);
        this.id = (+this.id || 0);
        this.timestamp =  new Date(this.timestamp);
        this.jsonObjectObject = JSON.parse(this.jsonObject);
    }
}